website/src/examples/home-demo.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
Can we use `.tsx`?

Reply:
I don't know. Everything in js in Docusaurus right now